src/lumigo_tracer/auto_tag/auto_tag_event.py

```diff
@@ -48,13 +48,25 @@ def auto_tag(event: dict):
                     add_execution_tag(key, headers[key])
 
 
+class ConfigurationHandler(EventAutoTagHandler):
+    @staticmethod
+    def is_supported(event) -> bool:
```

Comment:
```suggestion
    def is_supported(event: dict) -> bool:
```
I think we should add enforcement for types also for this repo